system/jlib/jdebug.cpp

```diff
@@ -818,9 +739,208 @@ struct KERNEL_USER_TIMES {
 //    );
 
 
-typedef LONG (WINAPI *PROCNTQSI)(UINT,PVOID,ULONG,PULONG);
-typedef LONG (WINAPI *PROCNTQIP)(HANDLE,UINT,PVOID,ULONG,PULONG);
-typedef LONG (WINAPI *PROCNTGST)(LARGE_INTEGER*, LARGE_INTEGER*, LARGE_INTEGER*);
+typedef LONG(WINAPI *PROCNTQIP)(HANDLE, UINT, PVOID, ULONG, PULONG);
+
+static struct CNtKernelInformation
+{
+    CNtKernelInformation()
+    {
+        NtQueryInformationProcess = (PROCNTQIP)GetProcAddress(
+            GetModuleHandle("ntdll"),
+            "NtQueryInformationProcess"
+        );
+        GetSystemInfo(&SysBaseInfo);
+}
+
+    PROCNTQIP NtQueryInformationProcess;
+
+    SYSTEM_INFO       SysBaseInfo;
+
+} NtKernelFunctions;
+#endif
+
+
+//===========================================================================
+
+#ifdef _WIN32
+static __uint64 ticksToNs = I64C(100);  // FILETIME is in 100ns increments
+#else
+static __uint64 ticksToNs = I64C(1000000000) / sysconf(_SC_CLK_TCK);
+#endif
+
+
+#ifdef _WIN32
+inline unsigned __int64 getFILETIME(const FILETIME & value)
```

Comment:
Minor, there is already a Windows:
```
BOOL GetFileTime(
  HANDLE     hFile,
  LPFILETIME lpCreationTime,
  LPFILETIME lpLastAccessTime,
  LPFILETIME lpLastWriteTime
);
```
Perhaps this could be named something a little different ?